lib/circles/create.js

```diff
@@ -7,9 +7,9 @@ module.exports = function(req, res) {
         mission: req.body.mission || null,
         aim: req.body.aim || null,
         expectationsForMembers: [],
-        members: [req.user.id],
+        members: [req.user.userId],
         invited: req.body.invited || [],
-        contactPerson: req.user.id,
+        contactPerson: req.user.userId,
```

Comment:
I'm not sure I like this pattern. req.user.id adds plenty of context to the id, but when documenting it, it makes sense to be explicit.